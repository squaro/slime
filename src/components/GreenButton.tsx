import styled from 'styled-components';

const GreenButton = styled.button`
  padding: 20px 72px;
  border-radius: 10px;
  box-shadow: 3px 3px 6px rgba(0,0,0,0.3);
  background-color: #0DBB13;
  outline: none;
  border: 0 none;
  font-weight: bold;
  font-size: 16px;
  color: #FFFFFF;
  text-transform: uppercase;

  &:active {
    background-color: #0DAA12;
    box-shadow: 3px 3px 6px rgba(0,0,0,0.2);
  }
`;

export default GreenButton;
