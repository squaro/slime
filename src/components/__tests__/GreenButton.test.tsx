import { render } from '@testing-library/react';
import GreenButton from '../GreenButton';

// TODO: Test hover case
describe('GreenButton', (): void => {
  it('should load green button component', (): void => {
    // Act
    render(<GreenButton />);

    // Assert
    expect(document.body).toMatchSnapshot();
  });
});
