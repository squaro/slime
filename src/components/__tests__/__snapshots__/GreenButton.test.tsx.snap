// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GreenButton should load button 1`] = `
<body>
  <div>
    <button
      class="sc-bdfBwQ jgMquD"
    />
  </div>
</body>
`;
